app/src/main/assets/native/injectionScript.js

```diff
@@ -0,0 +1,15 @@
+(() => {
+    const scripts = [
+        '/native/nativeshell.js',
+        '/native/EventEmitter.js',
+        document.currentScript.src.concat('?', Date.now())
+    ];
+    for (const script of scripts) {
+        const scriptElement = document.createElement('script');
+        scriptElement.src = script;
+        scriptElement.charset = 'utf-8';
+        scriptElement.setAttribute('defer', '');
+        document.body.appendChild(scriptElement);
```

Comment:
@nielsvanvelzen I changed this back, please see my comment in the commit.

Reply:
I'd still put it in the head tbh

Reply:
But why? I don't see a reason for changing the current behavior of Jellyfin Web.

Reply:
Scripts belong in the <head> element, although in HTML5 that isn't required anymore. Not a huge issue, don't have a strong preference for it either.

Reply:
Just as a comment but actually `jellyfin-web` does inject the scripts into head tag
https://github.com/jellyfin/jellyfin-web/blob/0dde17fbd7b8318269de53784543cb2981715dce/src/components/castSenderApi.js#L22

Reply:
Interesting, webpack puts the bundle into the body though..